test/SlowTests/Issues/RavenDB-10493.cs

```diff
@@ -33,29 +33,32 @@ public void CanTranslateDateTimeMinValueMaxValue()
 
                 using (var session = store.OpenSession())
                 {
+                    var dateTimeUtcNow = DateTime.UtcNow;
+                    var dateTimeToday = DateTime.Today;
+                    
                     var query = from x in session.Query<Article>()
                                 let test = 1
                                 select new
                                 {
                                     x.DateTime,
                                     DateTimeMinValue = DateTime.MinValue,
                                     DateTimeMaxValue = DateTime.MaxValue,
-                                    DateTimeUtcNow = DateTime.UtcNow,
-                                    DateTimeToday = DateTime.Today
+                                    DateTimeUtcNow = dateTimeUtcNow,
+                                    DateTimeToday = dateTimeToday
                                 };
 
-                    RavenTestHelper.AssertEqualRespectingNewLines(@"declare function output(x) {
+                    RavenTestHelper.AssertEqualRespectingNewLines(@"declare function output(x, $p0, $p1) {
 	var test = 1;
-	return { DateTime : x.DateTime, DateTimeMinValue : new Date(-62135596800000), DateTimeMaxValue : new Date(253402297199999), DateTimeUtcNow : ((date) => new Date(date.getUTCFullYear(), date.getUTCMonth(), date.getUTCDate(), date.getUTCHours(), date.getUTCMinutes(), date.getUTCSeconds(), date.getUTCMilliseconds()))(new Date()), DateTimeToday : new Date(new Date().setHours(0,0,0,0)) };
```

Comment:
1. The idea here was to test that we properly convert `DateTime.UtcNow / DateTime.Today` to JS so we don't want to have it as parameters
2. Let's modify just the assertion to what we talked about

Reply:
Fixed